### ISSUE TITLE
Error message with variable does not disappear (while it should)

### ISSUE THREAD

**Issue by mvarewyck** [merlinoa/shinyFeedback#4]
Thanks for the nice package!

When using a static error message text, everything works fine. However, when I let the error message depend on the input value (variable error message), it does not disappear when the condition of showing the error message is no longer met. A minimal example is included below.

```
library(shiny)
library(shinyFeedback)

shinyApp(
    ui = fluidPage(
        
        useShinyFeedback(),
        tags$h4("Type text until you reach min. required length of 5 characters"),
        
        textInput(inputId = "text", label = "Variable error message"),
        textInput(inputId = "text2", label = "Static error message")
    
    ),
    server = function(input, output, session) {
      
      # Error message does not disappear when error should be fixed
      observeEvent(input$text, {
            
            # Show message
            feedbackDanger(
                inputId = "text",
                condition = nchar(input$text) < 5,
                text = paste("Too short, currently only", nchar(input$text), "characters")
            )
            
          })
      
      observeEvent(input$text2, {
            
            # Show message
            feedbackDanger(
                inputId = "text2",
                condition = nchar(input$text2) < 5,
                text = paste("Too short")
            )
            
          })
      
    }
)
```

**Comment by merlinoa** [comment]
@mvarewyck thanks for pointing this out, and thanks for the nice reprex!  It is certainly a serious bug. It had to do with how `feedback()` was keeping track of the different feedbacks.  I think I resolved the bug with this commit: 7d1f15d91a2f8950bd563829418b4025e5bea49a

Please download the dev version using `devtools::install_github("merlinoa/shinyFeedback")` and please let me know if you are still experiencing any issues.

**Comment by merlinoa** [comment]
closing this as I believe it is resolved
